src/assets/css/_colors.scss

```diff
@@ -50,12 +52,16 @@ $twitter-color: #50abf1;
 .light-grey {
   background: $light-grey;
 }
-.light-red {
-  background: $light-red;
+
+.darkenBlue {
+  background-color: $darkenBlue;
 }
-.text-light-grey {
```

Comment:
Why are you deleting classes that we are using man? introduce new ones but dot delete them.